src/components/image-with-text-block/image-with-text-block.vue

```diff
@@ -21,9 +21,16 @@
         {{ title }}
       </h2>
       <div
+        v-if="body"
```

Comment:
Because this component is used in other places that don't use the new Dato block, and therefor still provide html.